fetch.bs

```diff
@@ -4593,6 +4593,31 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    using <var>request</var>.
   </ol>
 
+ <li>
+  <p>If <var>response</var>'s <a for=response>status</a> is 421, then:
+
+  <ol>
+   <li>
+    <p>If the ongoing fetch is <a for=fetch>terminated</a>, then:
+
+    <ol>
+     <li><p>Let <var>aborted</var> be the termination's aborted flag.
+
+     <li><p>If <var>aborted</var> is set, then return an <a>aborted network error</a>.
+
+     <li><p>Return a <a>network error</a>.
```

Comment:
I haven't followed the whole discussion, but is there a reason that 421 isn't just forwarded through? Surfacing it as a network error is opaque, and do will be difficult to debug / handle -- is there an underlying security issue here?

Reply:
I just followed the above 401 and 407 case handlings.
@annevk ,@yutakahirano, any thoughts?

Reply:
This could happen during the fetch initiated below if the initial fetch (that resulted in a 421) was explicitly aborted somehow. It seems safer to error in that case so aborting doesn't result in new information, but I'm not sure it matters a whole lot. I think I'd prefer keeping this as-is and revisiting if there are deployment issues. Note that implementations can still show the relevant information in DevTools and such.

Reply:
OK. I was assuming that the common case here would be that the fetch aborts for some reason, and the initiating code wants to automatically re-fetch based upon request details that it has buffered. I suppose it can do so based upon the network error, but 421 conveys a little bit of additional information -- that the request definitely hasn't changed any server-side state.